go/batch-submitter/flags/flags.go

```diff
@@ -194,6 +194,11 @@ var (
 			"mnemonic. The mnemonic flag must also be set.",
 		EnvVar: prefixEnvVar("PROPOSER_HD_PATH"),
 	}
+	SequencerBatchType = cli.StringFlag{
+		Name:   "sequencer-batch-type",
+		Usage:  "The type of sequencer batch to be submitted. Valid arguments are legacy or zlib.",
```

Comment:
If this is an optional flag we can also set `Value: "legacy"` so that it doesn't fail to start up